lnbits/core/static/js/extensions.js

```diff
@@ -1,43 +0,0 @@
-new Vue({
```

Comment:
was this file unnecessary?

Reply:
It was a leftover before the merge of  `Extensions` (`/extensions`) and `Manage Extensions` (`/install`)